components/Buttons/src/Theming/MDCButton+MaterialTheming.m

```diff
@@ -128,7 +128,18 @@ - (void)applyTextThemeWithScheme:(nonnull id<MDCContainerScheming>)scheme {
 }
 
 - (void)applyTextThemeWithColorScheme:(id<MDCColorScheming>)colorScheme {
-  [MDCTextButtonColorThemer applySemanticColorScheme:colorScheme toButton:self];
+  [self resetUIControlStatesForButtonTheming];
```

Comment:
Please do not change Active libraries as part of this change – this PR should only be adding deprecation warnings.

Reply:
Sure. Will separate in following PRs.